README.md

```diff
@@ -81,7 +81,7 @@ This project involves creating a console application that allows users to keep t
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] `Optional`
- [ ] 👋 If the live demo is not available please label it not available than it going to an empty link it looks none professional 🥂